### PR TITLE
Feature/add deep learning

### PR DESCRIPTION
**Pull request by 0x72D0**
## Description
Add the deep learning at start

### CODE REVIEW

File: launch_auv7.sh [hunk 1]
```diff
@@ -2,4 +2,11 @@ export ADRESS_IP=$(hostname -I | awk '{print $1}')
 export AUV=AUV7
 sudo chmod -R a+rwx "/home/sonia/ssd/config_filterchain"
 docker rm $(docker ps -a -q)
+
+#start deep learning
+source ~/ssd/catkin_ws/devel/setup.bash
+source ~/ssd/catkin_cv_bridge/install/setup.bash --extend
+
+roslaunch --wait proc_detection proc_detection.launch &
```
Comment (Kam0ss) [inline launch_auv7.sh:10]: This is really dangerous

File: launch_auv8.sh [hunk 2]
```diff
@@ -2,4 +2,11 @@ export ADRESS_IP=$(hostname -I | awk '{print $1}')
 export AUV=AUV8
 sudo chmod -R a+rwx "/home/sonia/ssd/config_filterchain"
 docker rm $(docker ps -a -q)
+
+#start deep learning
+source ~/ssd/catkin_ws/devel/setup.bash
+source ~/ssd/catkin_cv_bridge/install/setup.bash --extend
+
+roslaunch --wait proc_detection proc_detection.launch &
```
Comment (Kam0ss) [inline launch_auv8.sh:10]: Same comment than before

**Review by Kam0ss — COMMENTED** [review]
Too dangerous

**Comment by 0x72D0** [comment]
we gonna keep the launch as a separate script

**Pull request closed without merging**